tools/replace_code_block.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import sys
+from traceback import print_tb
+from typing import AnyStr
+
+status = 0
```

Comment:
I was going to change the status to 2 if there was a real error, but I don't know what to do. Maybe return the status instead I guess

Reply:
You could still `status |= result`. In the event both something was updated and there was an exception, then it'll exit with status code 3. Seems reasonable.

Also this might be a moot point with the idea of simply letting the exception kill the execution.